plugin.video.bfi/resources/lib/kodiutils.py

```diff
@@ -0,0 +1,106 @@
+# -*- coding: utf-8 -*-
+
+import json as json
+import logging
+
+import xbmc
+import xbmcaddon
+import xbmcgui
+
+# read settings
+ADDON = xbmcaddon.Addon()
+logger = logging.getLogger(__name__)
+
+ADDON_ID = ADDON.getAddonInfo("id")
+MEDIA_URI = "special://home/addons/{}/resources/media/".format(ADDON_ID)
+
+
+def art(image):
+    return ({
+        "thumb": image.get("data-img-800", ""),
+        "fanart": image.get("data-img-1440", ""),
+        "poster": image.get("src")
+    } if "src" in image else {
+        "thumb": image,
+        "fanart": image,
+        "poster": image
+    })
+
+def icon(image):
```

Comment:
You can use `xbmcaddon` to grab the icon. `xbmcaddon.Addon().getAddonInfo('icon')`